client/blocks/image-editor/image-editor-crop.jsx

```diff
@@ -102,7 +103,10 @@ class ImageEditorCrop extends Component {
 			} );
 		}
 
-		if ( aspectRatio !== newProps.aspectRatio ) {
+		if (
+			( aspectRatio !== newProps.aspectRatio ) ||
+			! this.state.wasCropUpdated
```

Comment:
Thanks 👍 this one is rather tricky to name, but I think this helps any future readers.